cl/citations/management/commands/import_citations_csv.py

```diff
@@ -38,9 +39,15 @@ def load_citations_file(options: dict) -> DataFrame | TextFileReader:
     end_row = None
 
     if options["start_row"] and options["end_row"]:
-        start_row = options["start_row"] if options["start_row"] > 1 else 0
+        start_row = options["start_row"] - 1 if options["start_row"] > 1 else 0
         end_row = options["end_row"] - options["start_row"] + 1  # inclusive
 
+    if options["start_row"] and not options["end_row"]:
+        start_row = options["start_row"] - 1 if options["start_row"] > 1 else 0
+
+    if options["end_row"] and not options["start_row"]:
+        end_row = options["end_row"]
```

Comment:
If you set the default value for these parameters to zero in your argparser, could you  delete all this?

Reply:
Yes, you are right, here i can reduce the code for the start row, for the end row we cannot pass 0 to nrows because it would not return any row, although it can also be reduced